packages/web-components/fast-element/src/templating/repeat.ts

```diff
@@ -316,9 +316,9 @@ export class RepeatDirective<TSource = any> extends HTMLDirective {
      * @param options - Options used to turn on special repeat features.
      */
     public constructor(
-        private itemsBinding: Binding,
-        private templateBinding: Binding<TSource, SyntheticViewTemplate>,
-        private options: RepeatOptions
+        public readonly itemsBinding: Binding,
+        public readonly templateBinding: Binding<TSource, SyntheticViewTemplate>,
+        public readonly options: RepeatOptions
```

Comment:
@EisenbergEffect I wanted to get your attention on this part in particular because it affects fast-element. I made these public so that the SSR renderer can inspect the necessary data to render. If there is another approach to getting this data into the SSR renderer let me know and I can adjust this.

Reply:
I'll make reviewing the full PR a high priority for first thing next week when I'm back. But these specific changes here seem fine to me.